src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -561,10 +561,12 @@ void uct_rc_mlx5_iface_commom_clean_srq(uct_rc_mlx5_iface_common_t *mlx5_common_
         cqe = uct_ib_mlx5_get_cqe(mlx5_cq, pi);
         if ((ntohl(cqe->sop_drop_qpn) & UCS_MASK(UCT_IB_QPN_ORDER)) == qpn) {
             idx = ntohs(cqe->wqe_counter);
-            seg = uct_ib_mlx5_srq_get_wqe(&mlx5_common_iface->rx.srq, idx);
-            seg->srq.free = 1;
-            ucs_trace("iface %p: freed srq seg[%d] of qpn 0x%x",
-                      mlx5_common_iface, idx, qpn);
+            if (mlx5_srq) {
+                seg = uct_ib_mlx5_srq_get_wqe(mlx5_srq, idx);
+                seg->srq.free = 1;
+                ucs_trace("iface %p: freed srq seg[%d] of qpn 0x%x",
```

Comment:
need to print iface, not a CQ pointer (or change the trace, but iface pointer would be more useful, imo)